### PR TITLE
Corrects an oversight in using file URLs section

### PR DESCRIPTION
**Pull request by lamagnifica**
@srpearce @catong @pdesjardins 
Shelby had a question about sizes for uploaded files, so I was looking at this section and noticed an odd line or two. 

other tweaks made also. See http://edx.readthedocs.org/projects/edx-partner-course-staff/en/latest/course_assets/course_files.html#file-size

### CODE REVIEW

File: en_us/shared/course_assets/course_files.rst [hunk 1]
```diff
@@ -78,30 +77,23 @@ The **Files & Uploads** page refreshes to show the uploaded file.
...
 ********************************************
 
-After you upload a file, you can use the URLs listed for it. On the **Files &
-Uploads** page, the **URL** column lists a Studio URL and web URL for each
-file.
-
-Use the web URL if you want to perform either of the following actions.
-Add a file or image to a bulk email message...
-Provide a link to the file or image from outside the course.
+To use an uploaded file for your course, you add a reference to one of the URLs
```
Comment (pdesjardins) [inline en_us/shared/course_assets/course_files.rst:80]: I'm not overly worried about long lines, but do these need the Sublime line length plugin applied to them?

**Comment by pdesjardins** [comment]
Looks good! :+1:

File: en_us/shared/course_assets/course_files.rst (hunk 1 above)
Comment (lamagnifica) [inline en_us/shared/course_assets/course_files.rst:80]: um. I used the line length plugin. I think that this line in 79 characters long (+ trailing space). My line length is set to 80 -- is yours set differently?

Comment (pdesjardins) [inline en_us/shared/course_assets/course_files.rst:80]: Sorry, it must have been the view in my browser. I was misinterpreting the
line wrap I saw. Now it makes sense.

Peter

**Pull request merged**